### DISCUSSION TITLE
Les discussions sont maintenant fermées, SVP ouvrez un nouvel Issue même pour une question avec l'utilisation FRW, MERCI!

### DISCUSSION THREAD

**Discussion by anisite** [MTESSDev/FRW#1] [Annonces]
Les discussions sont fermées, ouvrir un issue pour du support https://github.com/MTESSDev/FRW/issues

**Comment by thinkphil** [comment]
Bonjour! Je travaille au Ministère de la cybersécurité et du numérique et nous regardons la possibilité d'utiliser cet outil pour recueillir de l'information à l'interne sous forme de sondages auprès du personnel de la fonction publique. J'aurais aimé en apprendra davantage sur les possibilités et limites de cet outil. Suis-je au bon endroit? :)

**Reply by LarcM7** [reply]
Bonjour @thinkphil,

Vous êtes au bon endroit et nous sommes heureux de votre intérêt envers FRW.

Nous pensons qu'FRW sera PARFAIT pour répondre votre besoin.

Pour débuter, nous invitons toujours nos utilisateurs à consulter la documentation disponible sur GitHub, le point de départ est le "README" de la [page FRW](https://github.com/MTESSDev/FRW#readme). 

Vous y trouverez un lien vers la présentation complète (PowerPoint) ainsi que les premières étapes pour expérimenter avec l'outil et débuter l'intégration.

Je vous invite ensuite à prendre contact avec moi pour toutes autres questions ou précisions. 

Merci
